pensieve/analysis.py

```diff
@@ -15,6 +16,18 @@
 from mozanalysis.utils import add_days
 
 from . import experimenter
+from pensieve.statistics import BootstrapMean, Statistic
+
+# todo: this should be moved to config once that PR lands
+# todo: find a better name
+@attr.s(auto_attribs=True)
+class AnalysisStep:
```

Comment:
This should get moved to `config.py` once that is merged and it probably deserves a better name?